_developer-files/connect/api/objects/form-properties/sources/saas/google-search-console-object.md

```diff
@@ -0,0 +1,85 @@
+---
+# -------------------------- #
+#     USING THIS TEMPLATE    #
+# -------------------------- #
+
+## NEED HELP USING THIS TEMPLATE? SEE:
+## https://docs-about-stitch-docs.netlify.com/reference/connect-templates/destination-form-property/
+## FOR INSTRUCTIONS & REFERENCE INFO
+
+
+# -------------------------- #
+#        CONTENT TYPE        #
+# -------------------------- #
+
+product-type: "connect"
+content-type: "api-form"
+form-type: "source"
+key: "source-form-properties-google-search-console-object"
+
+
+# -------------------------- #
+#        OBJECT INFO         #
+# -------------------------- #
+
+title: "Google Search Console Source Form Property"
+api-type: "platform.google-search-console"
+display-name: "Google Search Console"
+
+source-type: "saas"
+docs-name: "google-search-console" # This should be whatever integration.name is. Ex: LinkedIn Ads is linkedin-ads
+
+
+# -------------------------- #
+#      OBJECT ATTRIBUTES     #
+# -------------------------- #
+
+uses-start-date: true
+
+# Only source-specific attributes need to be listed here.
+# The following attributes are considered common,
+# and therefore don't need to be listed:
+# anchor_time, cron_expression, frequency_in_minutes, image_version, start_date 
+
+object-attributes:
+  - name: "site_urls"
+    type: "string"
+    required: true
+    description: |
+      The domains and websites that belong to your organization. Refer to the [{{ form-property.display-name }} documentation]({{ doc-link | append: "#set-up-console" }}) for instructions on how to set these URLs up in your {{ form-property.display-name }} account."
```

Comment:
How should these values be formatted? Should the user include the `https://` or leave it out?